plugins/controlboard/src/ControlBoardDriverPositionControl.cpp

```diff
@@ -222,8 +222,11 @@ bool GazeboYarpControlBoardDriver::checkMotionDone(const int n_joint, const int
 {
     if (!joints || !flags) return false;
     bool ret = true;
+    *flags = true;
     for (int i = 0; i < n_joint && ret; i++) {
-        ret = checkMotionDone(joints[i], &flags[i]);
+	bool done;
```

Comment:
Can you use 4 spaces instead of `tabs` ? Thanks!

Reply:
The editor I use is configured not to use `tabs` at all. I will check carefully again.